src/libs/ajax.js

```diff
@@ -235,6 +235,11 @@ const User = signal => ({
     }
   },
 
+  getTos: async () => {
+    const response = await fetchSam('tos/text', _.merge(authOpts(), { signal, method: 'GET' }))
```

Comment:
`GET` is the default
```suggestion
    const response = await fetchSam('tos/text', _.merge(authOpts(), { signal }))
```